packages/trees/backend/sirius-components-trees/src/main/java/org/eclipse/sirius/components/trees/description/TreeDescription.java

```diff
@@ -66,6 +76,8 @@ public final class TreeDescription implements IRepresentationDescription {
 
     private BiFunction<VariableManager, String, IStatus> renameHandler;
 
+    private Function<VariableManager, Object> objectProvider;
```

Comment:
This is not how it was expected to be named

Reply:
Changed